aspnetcore/razor-components/routing.md

```diff
@@ -14,21 +14,29 @@ By [Luke Latham](https://github.com/guardrex)
 
 Learn how to route requests in apps and about the NavLink component.
 
-[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/razor-components/common/samples/) ([how to download](xref:index#how-to-download-a-sample)). See the [Get started](xref:razor-components/get-started) topic for prerequisites.
```

Comment:
No more sample app?

Reply:
Not for now ... Blazor routing is different from the endpoint routing used by Razor Components, and the sample app is a Blazor app (by-design at this time, but subject to change by the time we reach RTM). For the time being, this topic will just describe the Razor Components approach. This was one of the things that Dan is guiding: I think he will want me to create a separate Blazor-based Routing topic for the Blazor node by the time we hit RTM.